src/lib/moderation.ts

```diff
@@ -67,6 +67,13 @@ export function describeModerationCause(
       description: 'You have hidden this post',
     }
   }
+  // @ts-ignore Temporary extension to the moderation system -prf
+  if (cause.type === 'muted-word') {
+    return {
+      name: 'Post hidden by muted word',
+      description: `You've chosen to hide a word or tag within this post.`,
```

Comment:
Oh bleh. We've got this thing rewritten in the moderation PR to be easy to localize but it's not easy here. Not a trivial change either IIRC.